src/pages/jeongaYeon/Login/Login.js

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Login.scss';
+import { useNavigate } from 'react-router-dom';
 
 const LoginJeonga = () => {
-  return <div>Login</div>;
+  const [id, setId] = useState('');
+  const [password, setPassword] = useState('');
+  const [valid, setValid] = useState(false);
+  const navigate = useNavigate();
+
+  const handleLogin = () => {
+    navigate('/main-jeonga');
+  };
+
+  const saveUserId = event => {
+    let userId = event.target.value;
+    setId(userId);
```

Comment:
```suggestion
    setId(event.target.value);
```
따로 변수로 뺄 필요없이 이렇게 바로 넣어줘도 되겠네요!